collections/_article/rendering-priorities-july-2023.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Godot Rendering Priorities: July 2023"
+excerpt: "The Godot rendering team continues to focus on stability, performance and usability"
+categories: ["progress-report"]
+author: Clay John
+image: /storage/blog/covers/rendering-priorities-july-2023.webp
+date: 2023-07-25 20:00:00
+---
+
+Now that 4.1 is out and in the hands of users, it is time for the rendering team to re-evaluate our priorities and set our sights on the future of Godot development. Overall, our priorities are not changing much from what we identified in [our April 2023 progress report](https://godotengine.org/article/rendering-priorities-4-1/ ).
+
+As we anticipated, we were unable to finish all of our priorities in time for 4.1. In part this was due to the short timeframe (we made the post on April 18, and entered feature freeze on June 8), but it was also the result of spending more time fixing bugs than we planned (hooray for fixed bugs!) and a few key changes taking much longer than expected, notably: GLES3 3D shadows, and time slicing DirectionalLight3D shadows.
+
+## Support
+
+As a quick reminder before jumping into the meat of this article, Godot is a non-profit, open source game engine developed by hundreds of contributors in their free time, as well as a handful of part or full-time developers hired thanks to [generous donations from the Godot community](https://fund.godotengine.org/). A big thank you to everyone who has contributed [their time](https://github.com/godotengine/godot/blob/master/AUTHORS.md) or [their financial support](https://github.com/godotengine/godot/blob/master/DONORS.md) to the project!
+
+If you'd like to support the project financially and help us secure our future hires, you can do so using the [Godot Development Fund](https://fund.godotengine.org/) platform managed by [Godot Foundation](https://godot.foundation/). There are also several [alternative ways to donate](/donate) which you may find more suitable.
+
+## Priorities
+
+I want to address what I consider to be a failure of communication in the last rendering priorities post. The previous post was not clear about whether it was a roadmap for 4.1, or if it was a list of things that were important to the rendering team. While the content of the post said not to expect everything to be done for 4.1, the title made it sound like everything on the list was planned to be finished for 4.1. My intention in writing the post was not to promise anything for 4.1, but rather to highlight things that the rendering team is currently working on (and it just so happened that 4.1 would be the next release). Going forward, I hope to be more clear that the list of priorities is not tied into any specific release.
+
```

Comment:
I think one thing we should add here, something we often overlook, is just because we think we can finish the work in time, still doesn't mean it will be included.

Each and every feature we work on needs to go through our approval process and issues may be raised by the community that weren't previously identified and prevent work to be included in a release until further work is completed.

In essence, we can never promise a feature for a particular release, we can only do our best. This is FOSS.